tests/integration/goldens/asset/samples/generated_samples/cloudasset_generated_asset_v1_asset_service_batch_get_assets_history_sync.py

```diff
@@ -35,6 +35,7 @@ def sample_batch_get_assets_history():
 
     # Initialize request argument(s)
     request = asset_v1.BatchGetAssetsHistoryRequest(
+        parent="*",
```

Comment:
I'm not familiar with this syntax - is this asterisk something that's typical for this service?

Reply:
The * wouldn't actually work if passed to the service. The snippetgen logic pulls the first resource pattern out of the protos, and in some uncommon cases it is just a `*` rather than some more specific pattern like `projects/{project}`. The `*` pattern creates problems for generating resource helper methods and tests too  - see #701